feature/AOS-4875 mattermost notifications (#34)
Skatteetaten/aurora: docs/documentation/aurora-config.md

| segment             |          |                | segment      | The segment the application exist in.                                                                                                                                                                                                             |
| message             |          |                | message      | An message that will be added to the ApplicationDeployment CRD.                                                                                                                                                                                   |

### Notifications

> Today we have an integration with mattermost for notification messages when an application or env has been deployed. In order to use this feature

Reviewer: ```suggestion
Get notification messages when an application or environment/namespace has been deployed. 

#### Mattermost

In order to use this feature
```

one has to use the channelId, which is not the same as the channel name. The channelId can be retrieved by going to a channel, then hitting view
info in the channel header. At the bottom of the dialog box you will find a greyed out channel id.

| path                                          | default | description                                                                                                |
| --------------------------------------------- | ------- | ---------------------------------------------------------------------------------------------------------- |
| `notification/mattermost/<channelId>`         |         | Simplified config for enabling a mattermost notification for the given channelId. This requires a boolean. |

> | `notification/mattermost/<channelId>/enabled` | true    | Set to false to disable notification for the given channel.                                                |

Reviewer 2: Jeg tror jeg ville skrevet noe her om at, denne kan brukes hvis man vil disable notifications i noen miljøer men tilate det i alle andre? Eller er det egentlig unødvendig her. Jeg blir ikke enig med meg selv om hvor detaljert dette skal være.
